package.json

```diff
@@ -13,10 +13,11 @@
   },
   "scripts": {
     "start": "nodemon -e ts,tsx,json,js,jsx --watch ./src --ignore ./dist --exec 'yarn build'",
-    "build": "yarn run clean && yarn run build:cjs && yarn run build:esm && yarn run build:types",
+    "build": "yarn run clean && yarn run build:cjs && yarn run build:esm && yarn run build:types; yarn copy-abi",
     "build:cjs": "tsc --project tsconfig.build.json --module commonjs --outDir ./dist/cjs --removeComments --verbatimModuleSyntax false && echo > ./dist/cjs/package.json '{\"type\":\"commonjs\"}'",
     "build:esm": "tsc --project tsconfig.build.json --module es2015 --outDir ./dist/esm && echo > ./dist/esm/package.json '{\"type\":\"module\",\"sideEffects\":false}'",
     "build:types": "tsc --project tsconfig.build.json --module esnext --declarationDir ./dist/types --emitDeclarationOnly --declaration --declarationMap",
+    "copy-abi": "abi=utils/abi/contracts; for dir in cjs esm; do dstdir=\"./dist/${dir}/${abi}\"; mkdir -p \"${dstdir}\"; cp ./src/${abi}/*.json \"${dstdir}\"; done",
```

Comment:
nb. I'd rather make copy-abi work for a single output dir (i.e. `./dist/cjs` or `./dist/esm`). Unfortunately `yarn` is a bit limited here - i.e. it doesn't seem feasible to do this via `package.json` without calling externally to a separate script.

Reply:
Update: I was able to implement a workaround for this, and I think it's better than what was initially proposed: https://github.com/across-protocol/sdk-v2/pull/420/commits/4a21390c11e711c3201cba81592e95f607666e6c